src/collector/Backend.cpp

```diff
@@ -54,8 +55,7 @@ void Backend::clone_from(const Backend & other)
 
     std::memcpy(&m_stat, &other.m_stat, sizeof(m_stat));
     std::memcpy(&m_calculated, &other.m_calculated, sizeof(m_calculated));
-
-    m_read_only = other.m_read_only;
+    m_status_text = other.m_status_text;
```

Comment:
`status` defines a type of a problem and `status_text` clarifies it for admin. As in `cpp-worker` we do have separate status codes for different types of problem, we actually can switch to generating text status on demand.